### ISSUE TITLE
Protobuff message error when launching a lobby.

### ISSUE THREAD

**Issue by Philaeux** [ValvePython/dota2#17]
Recently, I find these 2 errors as soon as I start a private lobby. It is not stopping the normal execution of the program.

```
dazzar_bot         | [2016-12-05 23:08:49,646] ERROR Failed to parse: <EGCBaseClientMsg.EMsgGCPingRequest: 3001>
dazzar_bot         | [2016-12-05 23:08:50,156] ERROR Failed to parse: <EDOTAGCMsg.EMsgGCToClientSteamDatagramTicket: 7581>
```

**Comment by rossengeorgiev** [comment]
It's failing to locate the protobuf messages for those. Probably need to be manually mapped. Thanks

**Comment by Philaeux** [comment]
By "I start a private lobby", I wanted to say when I call `launch_practice_lobby()`, not at lobby creation.

**Comment by rossengeorgiev** [comment]
Yeah, the ticket is for connecting to the datagram relays in order to connect to the server. I had a quick glance and it is not obvious which messages are used exactly. Honestly, I don't see a point mapping them as the client can't connect to a game server etc.
